test/docfx.Snapshot.Tests/SamplesTest.SeedHtml/html/api-CatLibrary.Cat-2.html-q-cat.verified.html

```diff
@@ -6,6 +6,7 @@
       <meta name="title" content="Class Cat<T, K>
  | docfx seed website ">
       
```

Comment:
@yufeih any idea why these have new lines appended to them? I only see this in API generation, the conceptual articles are correct with a single line title.

Reply:
They are caused by the changes [here](https://github.com/dotnet/docfx/pull/9585/files#diff-8417c211817bef3705128787744619c69b49d70b90d69f71355314fd7137d08bR17) and isn't specific to API pages, e.g. [this one](https://github.com/dotnet/docfx/pull/9585/files#diff-c9fa1b2b50c6ae0bd4b1dbd143eae0966acf1eeb5bf399330688051efb124afa)